file_picker/uploads/file_pickers.py

```diff
@@ -14,7 +16,8 @@ class Meta(object):
 
     def save(self, commit=True):
         image = super(ImageForm, self).save(commit=False)
-        file_path = self.cleaned_data['file']
+        # Strip any directory names from the filename
+        file_path = os.path.basename(self.cleaned_data['file'])
```

Comment:
I just tried to investigate why I had to do this, and I'm still a little stumped. Basically, the sample project broke in Django 1.10 with a SuspiciousOperation when we try to save the image file. It complains that '/tmp/image.jpg' is outside of MEDIA_ROOT. It doesn't happen in Django < 1.10. I think it's related to this [issue](https://code.djangoproject.com/ticket/26644) which was closed. My fix above works for both 1.10 and 1.9 (and below), and I believe that it is the correct solution. It will not have any untoward effects on fields with `upload_to` set, as the model in the sample project has that attribute set and it works well.

The problem is that our tests don't cover these lines, so I will try to create a test which covers these lines, fails under 1.10 with the old code, and succeeds on all versions with the new code.